README.md

```diff
@@ -39,15 +39,14 @@ Below are instructions for setting up a local Freesound installation for develop
 
 3. Download the [Freesound development data zip file](https://drive.google.com/file/d/1c6w01tE4dIt8lEMMmK5aBEGV40oqe9vi/view?usp=share_link) (~7GB) and uncompress it inside `freesound-data`. You should get permission to download this file from Freesound admins. File structure should look like this:
 
-       freesound/
-       freesound/freesound-data/
-       freesound/freesound-data/avatars/
-       freesound/freesound-data/displays/
-       freesound/freesound-data/previews/
+       freesound-data/
+       freesound-data/avatars/
+       freesound-data/displays/
+       freesound-data/previews/
```

Comment:
It is not that this was wrong, but `freesound/` here was the repository folder I guess. In any case it is clearer without `freesound/` as this was confusing.